prboom2/src/r_draw.c

```diff
@@ -587,3 +593,17 @@ int R_GetFuzzPos()
 {
   return fuzzpos;
 }
+
+void R_ResetFuzzCol(int rows)
+{
+  // Make sure previous columns are drawn before changing fuzzpos
+  R_ResetColumnBuffer();
+
+  fuzzpos = (fuzzpos + (rows / fuzzcellsize)) % FUZZTABLE;
+}
+
+void R_NewFuzzCol(int x, int rows)
```

Comment:
I don't feel that this function name is quite right. It's always tricky to name this kind of function though. I'd probably go with `R_CheckFuzzCol` for a vague connection to `R_CheckPlane`, which also implies that something may or may not happen. I'd also change `rows` to `height` in these functions.

Reply:
Ah yeah I agree